### DISCUSSION TITLE
Configurable Notification Display Timer

### DISCUSSION THREAD

**Discussion by taima7700** [blish-hud/Community-Module-Pack#163] [Module Suggestions]
It would be amazing if this module allowed us to adjust the timer the notifications would be displayed on the screen.
currently if I get up to get a glass of water which takes only a couple of seconds, I could potentially be missing out on events notifications.

**Comment by raedan01** [comment]
I'd love this as well or also a way to click a button to view them again. it pops up and goes away before i have a chance to read them most of the time. also i moved the default position and I get the popup in 2 places

**Comment by Krojack** [comment]
Just installed this and wished there was a display time I could configure OR just leave it on screen till I click it away. One or both of those options would be amazing.

**Comment by eckhardslejs-sys** [comment]
Please extend the display duration
